makefile

```diff
@@ -513,9 +514,14 @@ $(buildDir)/output.%.coverage.html:$(buildDir)/output.%.coverage
 	$(gobin) tool cover -html=$< -o $@
 # end test and coverage artifacts
 
+clean-lobster:
+	rm -rf $(lobsterTempDir)
+
+update-lobster: clean-lobster
+	EVGHOME=$(EVGHOME) LOBSTER_TEMP_DIR=$(lobsterTempDir) $(shell pwd)/scripts/update-lobster.sh
```

Comment:
You can just use `$(EVGHOME)/scripts/update-lobster.sh` rather than invoking `$(shell pwd)` again (every call to shell creates a new process, so don't do it more than necessary).